.github/actions/prepare-frontend/action.yml

```diff
@@ -5,7 +5,7 @@ runs:
     - name: Prepare Node.js
       uses: actions/setup-node@v3
       with:
-        node-version: 14.x
+        node-version: 16.10.0
```

Comment:
Lots of people reported that they need to use exactly version `16.10.0` to not crash on their CI.

Reply:
I'd maybe add a comment in the code here so we don't forget about this and change it by mistake.
If you decide to do that, please commit with `[ci skip]` and I'll force-merge it.

Reply:
done!